challenges/objects-arrays.js

```diff
@@ -89,6 +156,8 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+zooAnimals.map(function (zooAnimals){
```

Comment:
This problem is asking you to use the forEach() method. You are using the map() method, but in this case all you need to do is to change this from map() to forEach() and you'll be set.